### DISCUSSION TITLE
how to plot day in x axis, time in y axis and a heatmap plot in python for the values as obained in the figure?

### DISCUSSION THREAD

**Discussion by saikatghosh90** [community/community#39249] [Other Feature Feedback, Questions, & Ideas]
how to plot day in x axis, time in y axis and a heatmap plot in python for the values as obtained in the figure?

![question image](https://user-images.githubusercontent.com/23078939/202196941-5c294ce5-fd3a-4b8c-8989-caddb8d454d4.png)
[question data .xlsx](https://github.com/community/community/files/10022221/question.in.stackoverflow.xlsx)

**Comment by jfullstackdev** [comment]
not exactly but maybe this 

https://www.geeksforgeeks.org/graph-plotting-in-python-set-1/

then you can add your own customizations

**Reply by saikatghosh90** [reply]
these are very basic graphs buddy

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @saikatghosh90, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. You've provided the dataset but what source code are you currently using to create the graph in your original post (or is that what you desire)? I've gone ahead and moved it for you. Good luck!

**Comment by saikatghosh90** [comment]
many thanks for doing that.. i was not able to find an appropriate category
